examples/typescript/actors/http-echo/README.md

```diff
@@ -0,0 +1,96 @@
+# Typescript HTTP Echo
+
+This repository
+
+# Dependencies
+
+This demo relies on the following installed software
+
+| Name   | Description                                                                                     |
+|--------|-------------------------------------------------------------------------------------------------|
+| `wash` | [Wasmcloud Shell][wash] controls your [wasmcloud][wasmcloud] host instances and enables building actors |
+| `npm`  | [Node Package Manager (NPM)][npm] which manages packages for for the NodeJS ecosystem                  |
+| `node` | [NodeJS runtime][nodejs] (see `.nvmrc` for version)                                                       |
+
+[wash]: https://github.com/wasmCloud/wasmCloud/tree/main/crates/wash-cli
+[node]: https://nodejs.org
+[npm]: https://github.com/npm/cli
+
+# Get started
+
+## Install NodeJS dependencies
+
+If you have the basic dependencies installed, you can install NodeJS-level dependcies:
+
+```console
+npm install
+```
+
+## Start a wasmcloud host
+
+To start a wasmcloud host you can use `wash`:
+
+```console
+wash up
+```
+
+This command won't return (as it's the running host process), but you can view the output of the host.
+
+## Build the actor component
+
+To build the [actor component][wasmcloud-actor-component]:
+
+```console
+npm build
+```
```

Comment:
Could we set up the `wasmcloud.toml` to run `npm build` instead? That way you could use `wash build` here, and leave a note that it's just running `npm build` behind the scenes

Reply:
I see further in the package.json that the npm scripts do encapsulate `wash build`, I just think it would be very nice to have wash be the agent that builds all the thigns

Reply:
> Could we set up the wasmcloud.toml to run npm build instead? That way you could use wash build here, and leave a note that it's just running npm build behind the scenes

Ah sure! I was going to wait until we had your change in wash, but I can change this early.

> I see further in the package.json that the npm scripts do encapsulate wash build, I just think it would be very nice to have wash be the agent that builds all the thigns

Agreed